__tests__/app.test.js

```diff
@@ -152,3 +153,81 @@ describe("GET /api/users", () => {
     });
   });
 });
+
+/* ***************GET ARTICLE BY ID WITH COMMENT COUNT****************/
+
+describe("GET /api/articles/:article_id returns correct article", () => {
+  test("endpoint returns correct article", async () => {
```

Comment:
This test doesn't really reflect what's being outlined in the description. Please either rephrase the description to say we're testing for the comment count specifically, or expand the test to reflect the entire article.

Reply:
Please also do some sad path tests for what could happen with getArticleById - what if the id they pass doesn't exist? Or is of the wrong data type?